auth0/auth0-deploy-cli
# Unmark sandbox_version as readonly

### 🔧 Changes

Previously, there was a issue about sandbox_version Import. https://github.com/auth0/auth0-deploy-cli/issues/69 
So mark sandbox_version to readonly, and remove sandbox_version in export.

Now seems to have been resolved that issue. Fix so that sandbox_version is exported again.


### 📚 References
Fix: https://github.com/auth0/auth0-deploy-cli/issues/682

### 🔬 Testing

Following steps
1. Check the Node.js runtime version in tenant advanced settings
2. Export settings with a0deploy
3. Check to appears `sandbox_version` in tenant.json
4. Compare 1 and 3 step's Node version


### 📝 Checklist
